src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Ui create messages that are being display to user.
+ */
+public class Ui {
+
+    /**
+     * Returns exit message when user exit the programs.
+     *
+     * @return Exit message when user exit the programs.
+     */
+    public String showExit() {
+        String bye = "Bye. Hope to see you again soon!";
+        return bye;
+    }
+
+    /**
+     * Returns task added to the list.
+     *
+     * @param task Task that is added.
+     * @param taskList The list of tasks.
+     * @return Task added to the list.
+     */
+    public String showAddTask(Task task, TaskList taskList) {
```

Comment:
Maybe an alternative to `show` would be better, as this is not printing (and thus showing) anything.